app/models/service/utilities.rb

```diff
@@ -0,0 +1,20 @@
+module Service
```

Comment:
I think this is the way to go, however place it on `app/services/` instead to follow (unspoken..) [conventions from core](https://github.com/theforeman/foreman/tree/develop/app/services), also 'Utility' (singular)